src/System.Drawing.Common/tests/Printing/PrinterResolutionTest.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using Xunit;
+
+namespace System.Drawing.Printing.Test
+{
+    public class PrinterResolutionTest
+    {
+        [Fact]
+        public void Default_Ctor()
+        {
+            PrinterResolution pr = new PrinterResolution();
+            Assert.Equal(PrinterResolutionKind.Custom, pr.Kind);
+        }
+
+        [Theory]
+        [InlineData(PrinterResolutionKind.High)]
+        [InlineData(PrinterResolutionKind.Medium)]
+        [InlineData(PrinterResolutionKind.Low)]
+        [InlineData(PrinterResolutionKind.Draft)]
+        [InlineData(PrinterResolutionKind.Custom)]
+        public void Set_Kind(PrinterResolutionKind kind)
+        {
+            PrinterResolution pr = new PrinterResolution();
```

Comment:
Oh! I thought there was an editorconfig file somewhere telling me to that this whole time!

Ignore me here